.github/workflows/update-errorcodes.yml

```diff
@@ -0,0 +1,39 @@
+name: Create PR for new error codes
+
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    defaults:
+      run:
+        working-directory: ./all-clients
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "18.x"
+      - run: |
+          cp package.json package.json.orig
+          npm run rebuild
+          if [ "$(wc -l < package.json.orig)" = "$(wc -l < package.json)" ]; then
```

Comment:
Checking if specifically lines were added by `npm run rebuild`. I've seen lines be reordered by `npm` in the past, so just looking for "changes" is not enough.